smartseq2_single_sample/ss2_single_sample.wdl

```diff
@@ -20,14 +20,14 @@ workflow SmartSeq2SingleCell {
   # ref index name
   File hisat2_ref_name
   File hisat2_ref_trans_name
-  # array of string represents featuretypes
   # samples
   String stranded
   String sample_name
   String output_name
   File fastq1
   File fastq2
-  call run_hisat2.RunHisat2Pipeline {
+
+  call run_hisat2.RunHisat2Pipeline as qc {
```

Comment:
Is `qc` and `data` as descriptive as we can be?  Is `qc_pipeline` or `run_qc` or `qc_subworkflow` any more helpful?  Something easier to understand when looking through metadata.  Changes look good to me otherwise 👍